### ISSUE TITLE
When using Mirador as viewer, there is some widget weirdness happening

### ISSUE THREAD

**Issue by blalbrit** [sul-dlss/exhibits#767]
### Actionable work:
 - [ ] for item embeds, use the default viewer not the configured one.

See:

![screenshot 2017-10-18 16 12 13](https://user-images.githubusercontent.com/796990/31746993-34d47574-b41f-11e7-8152-59b314a0d796.png)

**Comment by mejackreed** [comment]
It might be useful to provide some context here or a link? What is the expected behavior?

**Comment by blalbrit** [comment]
Good idea - I don't want to dive too deep on this, but here's what it looks like is happening:

- the version of Mirador used by the embed widget when building a feature page (like https://exhibits.stanford.edu/parker-2/feature/anne-s-test-page) seems to default to the full-height viewing experience, with the side-panel enabled, as if the embed image view were expected to be the same as the main viewer on an item show page.
- my expectation would be that a full-sized viewer wouldn't be used as the basis for the embed widget, but that different behavior would be used, given that this is a different viewing context
- I don't know if enabling Mirador as the main viewer (for item show pages) forces the embed widgets to use Mirador, or if it's possible to retain the "normal" behavior of a simple embed viewer. I would expect @ggeisler could guide us further here.

**Comment by ggeisler** [comment]
Yeah, my first thought is that we should always show the non-Mirador version of the viewer in the embed widget, even if the exhibit has the Mirador viewer enabled. There is a design for a dedicated Mirador viewer widget: https://github.com/sul-dlss/exhibits/issues/415. I think that would be sufficient if a curator wanted to show an item(s) in a Mirador viewer on a feature page.

Separately, I think this issue has surfaced a more general issue with showing the new, UV-based 
embed viewer in a widget. Here's the same embed widget used in an exhibit where the viewer is the default, non-Mirador viewer:

<img width="883" alt="new_exhibit_for_yale_-_spotlight_at_stanford" src="https://user-images.githubusercontent.com/101482/31748024-227dc014-b425-11e7-91fa-f1a9b4b42844.png">

It didn't look too bad until I selected the cog icon. The download and embed menus display fine.

**Comment by blalbrit** [comment]
@ggeisler - thanks for the comment Gary. I was wondering if you had thought about the UV behavior in the widget as well, and glad to see you have. It's a big enough change that I imagine the team had already considered it, given that it does throw a few hooks here and there, like:

https://exhibits.stanford.edu/lanciani/feature/testembed

![screenshot 2017-10-18 17 06 22](https://user-images.githubusercontent.com/796990/31748275-ed5fd348-b426-11e7-8c98-385cf949b6bc.png)

**Comment by blalbrit** [comment]
@ggeisler - yeah, that screen shot didn't translate well. The curve has to do with size (and Attrib overlay), which I meant to convey was slightly different from the old embed widget, but totally acceptable.

**Comment by ggeisler** [comment]
@blalbrit I see, thanks. 

The attribution overlay is there intentionally, I believe. When we made design recommendations as part of the UV work I think I suggested it not show by default (it should also be available through the Info panel) because it feels pretty insistent and dominant (especially when viewing multiple items in a row) but we decided to keep it, either because of technical or policy reasons, I forget exactly.

I don't think we're done with work the UV embed viewer, so we can and should spend more time looking at how we might improve the appearance of it when used in contexts like a Spotlight widget.
